.env

```diff
@@ -0,0 +1,4 @@
+PORT=443
+HOST=0.0.0.0
```

Comment:
This should be in a .env.development file.
In production you'll use a different host address.